src/Ar/Hammers/Hammers.fun

```diff
@@ -166,3 +166,12 @@ END_FUNCTION_BLOCK
 		internal : SdInternal_typ;
 	END_VAR
 END_FUNCTION_BLOCK
+
+{REDUND_ERROR} {REDUND_UNREPLICABLE} FUNCTION_BLOCK SimulationStatus (*Used to find the Simulation status for your project*) (*$GROUP=User,$CAT=User,$GROUPICON=User.png,$CATICON=User.png*)
+	VAR_OUTPUT
+		out :  BOOL; (**)

```

Comment:
Out doesnt make the most sense, maybe isSimulation. Also there is no way of knowing if it has gotten a result yet or is still busy? I think this returns instantly in the INIT but not necessarily in the CYCLIC (it might though)

Reply:
This has been changed to a function eliminating the need for an output variable.